### ISSUE TITLE
Style installation lessons appendix page

### ISSUE THREAD

**Issue by 01zulfi** [TheOdinProject/theodinproject#2582]
related to #2506 

### PR Instruction

Add this line somewhere in this issue's PR: "closes #XXXX". Replace the X's with this issue's id (2582)

### Task:

~~Create a new html file in app/views directory.~~
~~Add a webpage title and heading called Appendix.~~
~~Loop over the lessons and display titles of lesson with `installation_lesson` as true. Also add a link to the lesson.~~

Above tasks already done.

1. Navigate to this file `app/view/lessons/installation_lessons/index.html.erb`
2. Give the webpage a title called "Installation Lessons Appendix"
3. Add the `text-center` class to the page heading
4. Wrap the link within the `li` tag here with a `p` tag: 
~~~ruby
  <ul>
    <% @lessons.each do |lesson|%>
     <li><%= link_to lesson.title, path_course_lesson_path(lesson.path, lesson.course, lesson) %></li>
    <% end %>
  </ul>
  ~~~
5. Give these classes to the `p` tag: `text-1xl text-center m-3 text-gray-600`
6. Add these classes to the link like so:
~~~ruby
 <%= link_to lesson.title, path_course_lesson_path(lesson.path, lesson.course, lesson), class: "underline hover:no-underline hover:text-gray-500" %>
 ~~~
 
 What the page will look like after the changes: 
 
 
![image](https://user-images.githubusercontent.com/85733202/151981404-e259db6a-e08f-4411-99a9-c7c8d3f49ae9.png)

### Note:

In case, there's a better styling choice, try to implement it with a tailwind class. If that's not enough, create a new styles file called `installations_lessons.scss` in this directory: `app/assets/stylesheets/lesson`.

**Comment by JJJ4032002** [comment]
Can I take it ?

**Comment by 01zulfi** [comment]
@JJJ4032002 Yes, thank you! :rocket:

**Comment by JJJ4032002** [comment]
@O1zulfi hi i tried running Odin locally and ran rspec in the command line. It says in the guide that all test should be green but several test are failing by shows F. Is it normal or am I doing something wrong.

**Comment by 01zulfi** [comment]
@JJJ4032002 Unfortunately, I'm unable to help with that. You should ask in the discord chat 😄

**Comment by JJJ4032002** [comment]
Ok thanks

**Comment by JJJ4032002** [comment]
@01zulfi Hi again it seems a very stupid question to ask but i have served the site locally but where to preview the changes that i have made. Thanks again

**Comment by 01zulfi** [comment]
@JJJ4032002 Thanks for asking. Definitely an oversight on my part by not linking in the issue.

After running the local server, you should be able to see this page at http://localhost:3000/lessons/installation_lessons

**Comment by 01zulfi** [comment]
resolved with #2599
